src/resources/lib/abstract-resource.js

```diff
@@ -954,6 +976,49 @@ module.exports = class AbstractResource extends Restypie.Resources.AbstractCoreR
   }
 
   /**
+   * Authenticates the requested fields
+   *
+   * @param bundle
+   * @param fields
+   * @returns {Promise.<TResult>}
+   */
+  authorize(bundle, fields) {
```

Comment:
This deals with get/patch/post fields and the permission required is obtained through `getPermissions`